diff --git a/node_modules/@irsdk-node/native/lib/irsdk_utils.cpp b/node_modules/@irsdk-node/native/lib/irsdk_utils.cpp
index ef91e3b..f807651 100644
--- a/node_modules/@irsdk-node/native/lib/irsdk_utils.cpp
+++ b/node_modules/@irsdk-node/native/lib/irsdk_utils.cpp
@@ -338,7 +338,7 @@ unsigned int irsdk_getBroadcastMsgID()
 
 void irsdk_broadcastMsg(irsdk_BroadcastMsg msg, int var1, int var2, int var3)
 {
-	irsdk_broadcastMsg(msg, var1, MAKELONG(var2, var3));
+	irsdk_broadcastMsg(msg, var1, static_cast<int>MAKELONG(var2, var3));
 }
 
 void irsdk_broadcastMsg(irsdk_BroadcastMsg msg, int var1, float var2)
diff --git a/node_modules/@irsdk-node/native/src/irsdk_node.h b/node_modules/@irsdk-node/native/src/irsdk_node.h
index bc39c64..6567d3b 100644
--- a/node_modules/@irsdk-node/native/src/irsdk_node.h
+++ b/node_modules/@irsdk-node/native/src/irsdk_node.h
@@ -32,12 +32,12 @@ private:
     Napi::Value __GetTelemetryTypes(const Napi::CallbackInfo &info);
     Napi::Value GetTelemetryVar(const Napi::CallbackInfo &info);
 
-    bool iRacingSdkNode::GetTelemetryBool(int entry, int index);
-    int iRacingSdkNode::GetTelemetryInt(int entry, int index);
-    float iRacingSdkNode::GetTelemetryFloat(int entry, int index);
-    double iRacingSdkNode::GetTelemetryDouble(int entry, int index);
-    Napi::Object iRacingSdkNode::GetTelemetryVarByIndex(const Napi::Env env, int index);
-    Napi::Object iRacingSdkNode::GetTelemetryVar(const Napi::Env env, const char *varName);
+    bool GetTelemetryBool(int entry, int index);
+    int GetTelemetryInt(int entry, int index);
+    float GetTelemetryFloat(int entry, int index);
+    double GetTelemetryDouble(int entry, int index);
+    Napi::Object GetTelemetryVarByIndex(const Napi::Env env, int index);
+    Napi::Object GetTelemetryVar(const Napi::Env env, const char *varName);
 
     bool _loggingEnabled;
     char* _data;
